backend/api/Models/WbsModel.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections.Generic;
+
+namespace Api.Models
+{
+    public class WbsModel
+    {
+        /// <summary>
+        /// A status code within SAP that is set if the WBS is "closed" -> is not allowed to record hours in it.
+        /// </summary>
+        public const string ClosedForTimeWritingStatusCode = "NTWR";
+
+        public string? Code { get; init; }
+
+        public string? Description { get; init; }
+
+        public IList<string> ActiveStatusIds { get; init; } = new List<string>();
+
+        public bool IsOpenForTimeWriting
+        {
+            get
+            {
+                return !ActiveStatusIds.Contains(ClosedForTimeWritingStatusCode);
+            }
+        }
```

Comment:
```suggestion
        public bool IsOpenForTimeWriting => !ActiveStatusIds.Contains(ClosedForTimeWritingStatusCode);
```
Can simplify oneliners using expression body

Reply:
Fixed